languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10164,7 +10164,7 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
-      <rule id="ADJ_AVANT_OU_ARRIERE" name="adj avant ou arrière" default="temp_off">
+      <rule id="ADJ_AVANT_OU_ARRIERE" name="adj avant ou arrière">
```

Comment:
Could you put '\2' in single quotes? Could you make sure that's the case in each of the rules you're activating? Also, to have a uniform formatting, I think it would be best to replace all « guillemets » with 'single quotes'.